src/Search/Api/Elasticsearch/Command/BuildIndexCommand.php

```diff
@@ -67,7 +67,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             try {
                 $delete = $this->client->deleteIndex();
             } catch (Throwable $e) {
-                $logger->debug($e->getMessage(), $e->getTrace());
```

Comment:
I tried the first instance, and for some reason I was getting NULL back from elasticsearch, I can re-look into that, it would be the easiest. I also tried just doing an inspect/get on the index, but the size of the response was massive!